### DISCUSSION TITLE
SAHI Postprocessing methods

### DISCUSSION THREAD

**Discussion by patrontheo** [obss/sahi#1086] [Q&A] [answered]
Hello @fcakyon 
I saw that there are multiple postprocessing methods that can be used in SAHI (GreedyNMMPostprocess, LSNMSPostprocess, NMMPostprocess, NMSPostprocess), with some parameters to set (e.g. match_metric). I didn't find much explanation about those.
I was wondering how you selected the default parameters (GreedyNMMPostprocess, 'IOS' for match metric, threshold of 0.5 for matching). Did you try the different methods and saw that it was the one working best ? I couldn't find details about the postprocessing in the paper.
I think it would be great if you could provide some documentation about how each postprocessing method works and what it does.
Thanks a lot

**Comment by fcakyon** [comment, marked as answer]
Hey @patrontheo, if you aim for academic map score based bencmarks, NMS works best. In practical use cases NMS creates lots of duplicate and false positives so non-maximum-merging (NMM) works better. NMM is similar to NMS but instead of removing matched boxes, it merges them.
